Changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 20231025 - Rewards, Loyalties
+
+**Changes in models**
+- Removed `reward_base`, `reward_identity`, `reward_created` - these types were combined into the `reward` model
```

Comment:
as we have now `reward` and `reward_assignment` models, please update file: `scripts/md-tables.ts`
`['4_obj_reward_object', 'REWARDS-API-Reward-Object.md'],`  and `['4_obj_reward_assignment_object', 'REWARDS-API-Reward-Assignment-Object.md'],`

Reply:
Done.
Previously there were three types of Reward: Digital, Pay with Points and Material so I also adapted the description to the previous version (not use Coin Reward, Campaign Reward and Material Reward)

Coin -> Pay With Points
Campaign -> Digital
Material -> Material (the same)